OpenOversight/app/models/database.py

```diff
@@ -41,20 +41,22 @@ class Department(BaseModel):
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(255), index=True, unique=True, nullable=False)
     short_name = db.Column(db.String(100), unique=False, nullable=False)
+    state = db.Column(db.String(2), index=True)
```

Comment:
As in my comment on the other PR, we don't need an index on state (The existing index on name also doesn't make much sense to me)